client/src/header.css

```diff
@@ -0,0 +1,52 @@
+#head{
+    position: fixed;
+    display: inline;
+    margin-top: 25px;
+    margin-left: 0px;
+    width: 100vw;
+    height: 104px;
+    opacity: 1;
+}
+
+#camera{
+    align-content: right;
+    margin-top: 40px;
+    margin-bottom: 33px;
+    margin-right: 10vmin;
+    margin-left: 86vmin;
+    width: 48px;
+    height: 48px;
+    background: transparent url('./camera.png') 0% 0% no-repeat padding-box;
+    opacity: 1;
+}
+
+#insta{
+    padding-top: 0px;
+    justify-content: center;
+    display: inline;
+    margin-top: 25px;
+    margin-left: 14.2vmin;
+    vertical-align: top;
+    width: 289px;   
+    height: 71px;
+}
+
+#txt{
+    padding-left:10px;
+    justify-content: center;
+    vertical-align: text-bottom;
+    font: normal normal normal 64px/71px "./fonts/Palatino.ttf";
```

Comment:
try to write each property separately , for example font size font family. also this is not how we giive font family value

Reply:
okay